<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wSettings</class>
 <widget class="QMainWindow" name="wSettings">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wSettings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressources.qrc">
    <normaloff>:/rec/data/icons/iconSettings.svg</normaloff>:/rec/data/icons/iconSettings.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="gbxGeneral">
      <property name="title">
       <string>General</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="lblLanguage">
           <property name="text">
            <string>Language:</string>
           </property>
           <property name="buddy">
            <cstring>cobxLanguage</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cobxLanguage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMoreLanguages">
           <property name="text">
            <string>More languages...</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/rec/data/icons/iconAdd.svg</normaloff>:/rec/data/icons/iconAdd.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="lblTheme">
           <property name="text">
            <string>Theme:</string>
           </property>
           <property name="buddy">
            <cstring>cobxTheme</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cobxTheme">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="lblAuthor">
           <property name="text">
            <string>Author:</string>
           </property>
           <property name="buddy">
            <cstring>ledAuthor</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ledAuthor">
           <property name="placeholderText">
            <string>My nickname</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblOmsiPath">
           <property name="text">
            <string>OMSI main directory:</string>
           </property>
           <property name="buddy">
            <cstring>ledOmsiPath</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ledOmsiPath">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnOmsiPath">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/rec/data/icons/iconOpen.svg</normaloff>:/rec/data/icons/iconOpen.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbxAutosave">
      <property name="title">
       <string>Backup</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Backup interval:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbxAutosaveDuration">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::UpDownArrows</enum>
             </property>
             <property name="specialValueText">
              <string/>
             </property>
             <property name="suffix">
              <string notr="true"/>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="stepType">
              <enum>QAbstractSpinBox::DefaultStepType</enum>
             </property>
             <property name="value">
              <number>15</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="lblDiskUsage">
           <property name="text">
            <string>Backup disk usage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnOpenBackupFolder">
           <property name="text">
            <string>Open backup folder</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/rec/data/icons/iconOpen.svg</normaloff>:/rec/data/icons/iconOpen.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteAllBackups">
           <property name="text">
            <string>Delete all backups</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/rec/data/icons/iconToBin.svg</normaloff>:/rec/data/icons/iconToBin.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QGroupBox" name="gbxLogging">
        <property name="title">
         <string>Logging</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="lblLogfileMode">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Mode:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cobxLogfileMode"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnOpenLogfilePath">
           <property name="text">
            <string>Open logfile path</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/rec/data/icons/iconOpen.svg</normaloff>:/rec/data/icons/iconOpen.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbxUpdates">
        <property name="title">
         <string>Updates</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="lblAutoUpdateCheck">
             <property name="text">
              <string>Check automatically:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cobxAutoUpdateCheck"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnCheckForUpdates">
             <property name="text">
              <string>Check for updates now</string>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/rec/data/icons/iconDownload.svg</normaloff>:/rec/data/icons/iconDownload.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="gbsMiscellaneous">
      <property name="title">
       <string>Miscellaneous</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="1" colspan="2">
          <widget class="QPushButton" name="btnCreateDesktopShortcut">
           <property name="text">
            <string>Create desktop shortcut</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/rec/data/icons/iconLink.svg</normaloff>:/rec/data/icons/iconLink.svg</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="2" colspan="2">
          <widget class="QCheckBox" name="cbxConfirmDeletion">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Confirm deletion</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnResetSettings">
        <property name="text">
         <string>Reset settings</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/rec/data/icons/iconUndo.svg</normaloff>:/rec/data/icons/iconUndo.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnRestart">
        <property name="text">
         <string>Restart</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/rec/data/icons/iconCheck.svg</normaloff>:/rec/data/icons/iconCheck.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/rec/data/icons/iconQuit.svg</normaloff>:/rec/data/icons/iconQuit.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
